src/Stream.ts

```diff
@@ -61,6 +61,52 @@ export interface Emitter<A> {
   end(): void
 }
 
+export const fromAsyncIterable: {
+  <A, R>(iterable: AsyncGenerator<A, R>): Fx.Fx<Async.Async | Stream<A>, R>
+  <A>(iterable: AsyncIterable<A>): Fx.Fx<Async.Async | Stream<A>, unknown>
+} = <A>(iterable: AsyncIterable<A>): Fx.Fx<Async.Async | Stream<A>, unknown> => Fx.bracket(
+  Fx.sync(() => iterable[Symbol.asyncIterator]()),
```

Comment:
I hadn't thought about this before, but if `iterable` is an AsyncGenerator, does that mean this line will return the same iterator every time?  i.e. the resulting Fx will not be safely restartable?

Reply:
Hmm yeah, that's a good catch. I guess I'd need to swap for a similar type as returned by `toAsyncIterable`

Reply:
Updated https://github.com/briancavalier/fx/pull/5/commits/9d76eb7cf4537b7912997ec48d29492f92ac1b1e